### ISSUE TITLE
Feature request: PAUSE on layer for filament change

### ISSUE THREAD

**Issue by Originalas** [CrealityOfficial/CrealityPrint#57]
It would be nice if there would be a button/menu to insert a PAUSE macro on a selected layer. 
So we could do color prints by swapping filament and without editing G-Code file manually like in this video https://www.youtube.com/watch?v=KvGgrkUanTo

**Comment by SargFowler** [comment]
Indeed. Prusaslicer allows you to do this easily and will even suggest colour changes if it detects text on the top of a print.

**Comment by botolo** [comment]
I would loooooove this option! I hope it can be implemented. With Hueforge, changing filaments is becoming a must.

**Comment by protopopescu** [comment]
Yes, please add this option, as it would be very useful, and it's easy to implement.

**Comment by Domingo1987** [comment]
Yes, please. Do it. Hazlo
